include/mqtt/endpoint.hpp

```diff
@@ -6583,12 +6583,22 @@ class endpoint : public std::enable_shared_from_this<endpoint<Mutex, LockGuard,
                 ]
                 (buffer body, buffer buf, any session_life_keeper, this_type_sp self) mutable {
                     info.reason_code = static_cast<v5::puback_reason_code>(body[0]);
-                    process_puback_impl<puback_phase::properties>(
-                        force_move(session_life_keeper),
-                        force_move(buf),
-                        force_move(info),
-                        force_move(self)
-                    );
+                    if (remaining_length_ == 0) {
```

Comment:
// https://docs.oasis-open.org/mqtt/mqtt/v5.0/os/mqtt-v5.0-os.html#_Toc471282795
// If the Remaining Length is less than 4 there is no Property Length and the value of 0 is used.